Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:boron
+
+# Create app directory
+RUN mkdir -p /usr/src/dc-campaign-finance-watch
+WORKDIR /usr/src/dc-campaign-finance-watch
+
+# Install app dependencies
+COPY package.json /usr/src/dc-campaign-finance-watch/
+RUN npm install
+
+# Bundle app source
+COPY . /usr/src/dc-campaign-finance-watch
```

Comment:
yea, this docker file was the suggest for node dev https://nodejs.org/en/docs/guides/nodejs-docker-webapp/ 
I'm going to check for prod suggestions.